srtcore/congctl.cpp

```diff
@@ -167,7 +167,8 @@ class LiveCC: public SrtCongestionControlBase
         // packet = payload + header
         const double pktsize = (double) m_zSndAvgPayloadSize + CPacket::SRT_DATA_HDR_SIZE;
         m_dPktSndPeriod = 1000 * 1000.0 * (pktsize / m_llSndMaxBW);
-        HLOGC(mglog.Debug, log << "LiveCC: sending period updated: " << m_zSndAvgPayloadSize);
+        LOGC(mglog.Error, log << "LiveCC: sending period updated: " << m_dPktSndPeriod
```

Comment:
I believe it was temporary. Still, might be a new FA would be needed.

Reply:
Yes, this is a temporal log for testing. I will delete those after PR is finished.